cmd/juju/action/show_test.go

```diff
@@ -87,9 +87,14 @@ Arguments
 full:
   type: boolean
   description: take a full backup
+  default: true
 name:
   type: string
   description: snapshot name
+prefix:
+  type: string
+  description: prefix to snapshot name
+  default: ""
```

Comment:
I think we are now missing the case where there is no default, are we?

Reply:
I adjusted the [test actions](https://github.com/juju/juju/blob/1b2093b01b91e6020ac0084c2f6030bf194daf5c/cmd/juju/action/package_test.go#L80) in `package_test.go` so I think we're covered. We have a test for no default, a blank default and a default value

Reply:
See the "name" field above? :)

Reply:
yep, sorry I missed it.